test/test_autograd.py

```diff
@@ -2540,14 +2499,14 @@ def run_grad_and_gradgrad_checks(test_case, name, test_name, apply_method, outpu
     test_case.assertTrue(gradcheck(apply_method, input_variables, eps=1e-6, atol=PRECISION))
     if name in EXCLUDE_GRADGRADCHECK or test_name in EXCLUDE_GRADGRADCHECK_BY_TEST_NAME:
         return
-    grad_y = generate_gradoutput(output_variable, non_contiguous=True)
     gradgradcheck_precision_override = gradgradcheck_method_precision_override(test_name)
     if gradgradcheck_precision_override is not None:
         atol = gradgradcheck_precision_override['atol']
         rtol = gradgradcheck_precision_override['rtol']
-        test_case.assertTrue(gradgradcheck(apply_method, input_variables, grad_y, atol=atol, rtol=rtol))
+        test_case.assertTrue(gradgradcheck(apply_method, input_variables, None, atol=atol, rtol=rtol,
+                                           gen_grad_outputs_non_contig=True))
     else:
-        test_case.assertTrue(gradgradcheck(apply_method, input_variables, grad_y))
+        test_case.assertTrue(gradgradcheck(apply_method, input_variables, gen_grad_outputs_non_contig=True))
```

Comment:
nit: `non_contig_grad_outputs` would sound a bit better

Reply:
I like that more, will rename.